frontend/elmapp.config.js

```diff
@@ -1,10 +1,5 @@
 module.exports = {
     inline: true,
     historyApiFallback: true,
-    proxy:{
-        '/':{
-            target: 'http://localhost:8080',
-            secure: false
-        }
-    }
+    proxy: 'http://localhost:8080'
```

Comment:
update structure to newer version of create-elm-app (updated in https://github.com/Holmusk/elm-street/pull/103 / https://github.com/Holmusk/elm-street/pull/104)